test/validate/eia_test.py

```diff
@@ -42,15 +45,18 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
     [
         ("bf_eia923", 1_427_692, 1_427_692, 119_611),
         ("bga_eia860", 130_326, 130_326, 130_326),
-        ("boil_eia860", 74_086, 74_086, 74_086),
+        ("boil_eia860", 74_169, 74_169, 74_169),
         ("frc_eia923", 597_000, 244_415, 24_065),
         ("gen_eia923", None, 5_171_497, 432_570),
         ("gens_eia860", 523_563, 523_563, 523_563),
         ("gf_eia923", 2_687_345, 2_687_345, 230_149),
         ("own_eia860", 84_440, 84_440, 84_440),
-        ("plants_eia860", 185_553, 185_553, 185_553),
-        ("pu_eia860", 184_743, 184_743, 184_743),
-        ("utils_eia860", 119_366, 119_366, 119_366),
```

Comment:
The difference in rows here comes from the fact that some utilities aren't showing up in the utils_eia860 or pu_eia860 tables in later years. None of these new rows are from new utilities, just utilities that didn't report certain years. This is only a little annoying because it means that the utils_eia860 table will have a certain years of utilities that don't have all the standard information you'd find in the utils_eia860 table like address etc. We could potentially backfill. Not doing anything about it now, just wanted to flag!